src/components/organization/OrganizationSettings/OrganizationSettings.tsx

```diff
@@ -35,12 +57,25 @@ const transferOwnershipDetails =
 interface Props {
   /** Authenticated user. */
   user: Session["user"];
-  /** Organization ID. */
-  organizationId: Organization["rowId"];
+  /** Organization details. */
+  organization: OrganizationRow;
+  /** Customer information derived from the signed in user. */
+  customer: CustomerState | undefined;
+  /** Backfeed subscription products. */
```

Comment:
Generalized comment in case Backfeed ever gets renamed

```suggestion
  /** App subscription products. */
```